Source/Core/Core/DSP/DSPStacks.cpp

```diff
@@ -3,38 +3,44 @@
 // Licensed under GPLv2+
 // Refer to the license.txt file included.
 
-#include "Common/CommonTypes.h"
+#include "Core/DSP/DSPStacks.h"
 
+#include <cstddef>
+
+#include "Common/CommonTypes.h"
 #include "Core/DSP/DSPCore.h"
-#include "Core/DSP/DSPStacks.h"
 
 // Stacks. The stacks are outside the DSP RAM, in dedicated hardware.
 namespace DSP
 {
-static void dsp_reg_stack_push(int stack_reg)
+static void dsp_reg_stack_push(size_t stack_reg)
```

Comment:
Whoops, missed the `static`, thought they're the ones from the header (which do take `StackRegister` parameters). Nvm then.